pkg/agent/agent.go

```diff
@@ -1215,6 +1222,9 @@ func (i *Initializer) prepareOVSBridge() error {
 }
 
 func (i *Initializer) prepareOVSBridgeForVM() error {
+	if err := i.prepareOVSConfigForVM(); err != nil {
+		return err
+	}
```

Comment:
could we move `prepareOVSBridgeForVM` to platform specific files directly to avoid many short functions with variant names? The following may be less verbose and symmetrical.

```
// prepareOVSBridgeForK8sNode adds local port and uplink port to OVS bridge after OVS extension is enabled on HNSNetwork.
// This function deletes OVS bridge and HNS network created by Antrea on failure.
func (i *Initializer) prepareOVSBridgeForK8sNode() error {
    hnsNetwork, err := hcsshim.GetHNSNetworkByName(util.LocalHNSNetwork)
    ...
}

// agent_windows.go
func (i *Initializer) prepareOVSBridgeForVM() error {
    klog.InfoS("Performing OVS configuration", "hostIfName", i.nodeConfig.UplinkNetConfig.Name)
    hostIfName := i.nodeConfig.UplinkNetConfig.Name
    uplinkIfName := util.GenerateUplinkInterfaceName(hostIfName)
    ...
    return i.setOVSDatapath()
}
```

Reply:
Sounds good. I have updated it.